File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/comment/comment/direct/Java/C/s725559666_commented.c
#include <stdio.h>

int main() {
    int a, b, c;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    if (a < b && b < c && a < c) {
        printf("Yes\n");
    } else {
        printf("No\n");
    }
    return 0;
}
// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/inline/comment/direct/Java/Go/s725559666_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Split(bufio.ScanWords)
	var a, b, c int
	if scanner.Scan() {
		a, _ = strconv.Atoi(scanner.Text())
	}
	if scanner.Scan() {
		b, _ = strconv.Atoi(scanner.Text())
	}
	if scanner.Scan() {
		c, _ = strconv.Atoi(scanner.Text())
	}
	if a < b && b < c && a < c {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

// End of Code
